YellowNotes.Core/Utility/GetNotesConfig.cs

```diff
@@ -0,0 +1,22 @@
+using System.Linq;
+
+namespace YellowNotes.Core.Utility
+{
+    public class GetNotesConfig
+    {
+        public int TakeCount { get; set; } = 20;
+
+        public int SkipCount { get; set; } = 0;
+
+        public int[] CategoryIds { get; set; } = new int[0];
+
+        public bool IsCategorySelected(int? categoryId)
```

Comment:
dto shouldn't have any method, any logic. read property yes, but not method

Reply:
Should i create a static method in a separate class or an extension method?

Reply:
it looks like business logic, so put it to service